index.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Tribute Page</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <main id="main"> 
+        <div id="img-div">
+          <figure>
+          <img id="image" src="https://i0.wp.com/www.themarginalian.org/wp-content/uploads/2014/01/alanwatts.jpg?w=680&ssl=1">
+          <figcaption id="img-caption">Picture of Allan Watts</figcaption>
+          </figure>
+          <p id="tribute-info">He was a filosopher and a meditation expert.</p>
+         <a id="tribute-link" href="https://en.wikipedia.org/wiki/Lionel_Messi" target="_blank">Lionel messi history</a>         
+        </div>
+        <title id="title">Rest in Peace Alan Watts</title>
```

Comment:
I think it would be a good idea to remove this line because the title is only used in the `head` section and you already have one.